### PR TITLE
[NEW] Mainstage Speakers Conference Page

### PR DESCRIPTION
**Pull request by sidmohanty11**
This is still work in progress, 

- [x] Make infotiles component more generic
- [x] Above circular SVGs are needed to be added
- [x] On mobile screen, wrap it to one speaker card component not 2
- [x] Use strapi cms to retrieve data

## Web

screencapture-localhost-3000-virtualconf-mainstage-2022-02-27-02_50_52

## Mobile
screencapture-localhost-3000-virtualconf-mainstage-2022-02-27-02_51_26

### CODE REVIEW

**Comment by sidmohanty11** [comment]
## Web View
screencapture-localhost-3000-virtualconf-mainstage-2022-02-27-20_27_45

## Mobile View
screencapture-localhost-3000-virtualconf-mainstage-2022-02-27-20_28_04

And the data is fetched from strapi, it has a Speakers Component now which possess the mentioned fields:
- id
- name (of the speaker)
- content (a small description of speaker)
- imageUrl (PhotoUrl of the speaker)
- live (which is a boolean, so that we can conditionally render the live button and provide the confHref)
- ended (also a boolean, to disable out the speakers as described in the figma design)
- confHref (conference link)

## Strapi
Screenshot from 2022-02-27 20-30-32

@Sing-Li please check the model and suggest the changes

File: app/pages/_app.js [hunk 1]
```diff
@@ -7,11 +7,10 @@ import { initAuth } from '../components/auth/firebase';
 initAuth();
 
 function MyApp({ Component, pageProps }) {
-  return (
+  const getLayout = Component.getLayout || ((page) => <Layout menu={pageProps}>{page}</Layout>)
+  return getLayout(
```
Comment (Sing-Li) [inline app/pages/_app.js:11]: Are we sure this change doesn't make all pages not SSR and/or static gen-able?

**Review by Sing-Li — CHANGES_REQUESTED** [review]

**Comment by Sing-Li** [comment]
Please add an announcement or a banner (banner component?)   to the home page and link to this conference.    Please consult @demonicirfan if you need some minor  design help here.

File: app/pages/_app.js (hunk 1 above)
Comment (sidmohanty11) [reply app/pages/_app.js:11]: Yeah, actually the navbar doesn't go well with the theme so I thought of going with the design completely. Any suggestions of what should we do here?

**Comment by sidmohanty11** [comment]
Sure, let me consult with him with the design and I'll let you know

**Comment by Sing-Li** [comment]
Thanks for the updates.  This is a great foundation to build other parts of the virtual conference handling on.    I will go ahead and merge it.

**Pull request merged**